docs/docs/how-tos/nebari-upgrade.md

```diff
@@ -27,19 +99,15 @@ Always [backup your data](./manual-backup.md) before upgrading!
 
 ### Locate configuration file
 
-You may be deploying Nebari based on a local configuration file, or you may be using CI/CD workflows in GitHub or GitLab.
-Either way, you will need to locate a copy of your `nebari-config.yaml` configuration file to upgrade it (and commit back
-to your git repo in the CI/CD case).
+Locate your `nebari-config.yaml` configuration file.
 
-For CI/CD deployments, you will need to `git clone <repo URL>` into a folder on your local machine if you haven't done
-so already.
+For CI/CD deployments, you will need to `git clone <repo URL>` into a folder on your local machine if you haven't done so already.
 
 :::warning Breaking upgrades
 
 ### Rename existing Nebari URL
```

Comment:
Similar to my previous comment, this heading is not visible in the right-sidebar TOC, so I would suggest moving the warning inside the heading, something like:

```
	### Rename existing Nebari URL
	
	:::warning Breaking upgrades
	This will allow the existing cluster to remain alive in case it is needed, but the idea would be not to have it in use from now on.
	:::
	
	In the `nebari-config.yaml` for example:
	
	```yaml
	project_name: mynebari
	domain: nebari.myproj.com
	```
	could become:
	```yaml
	project_name: mynebari
	domain: nebari-old.myproj.com
	```
	Run the command `nebari deploy` using the existing (old) version of Nebari.

```
 
@ppwadhwa What do you think?